banbot.js

```diff
@@ -407,6 +455,41 @@ bot.on('message', async ( message ) => {
             else mongo.suspensionDue( id )
         })
     }
+    else if ( content.startsWith( '.comp' ) ) {
+        message.delete()
+        let target = message.mentions.members.array().shift()
+        let id = target ? target.id : null
+        if ( !target ) {
+            if ( content.split( ' ' ).length == 2 ) {
+                target = content.split(' ').pop()
+                id  = target.replace( /\D*/g, '' )
```

Comment:
If you do decide to add a `reason` field, you'll need to modify this code as well.

Reply:
Cpl as asked for no reason ( reason will be added to minor/mod/major fields